### PR TITLE
Development tracing improvements

### PR DESCRIPTION
**Pull request by timneutkens**
- Fix arg dep
- Fix benchmark prepare
- Add additional spans for development trace

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have helpful link attached, see `contributing.md`

## Documentation / Examples

- [ ] Make sure the linting passes

### CODE REVIEW

File: packages/next/build/webpack/plugins/profiling-plugin.ts [hunk 1]
```diff
@@ -99,17 +99,30 @@ export class ProfilingPlugin {
...
     compiler.hooks.compilation.tap(pluginName, (compilation: any) => {
+      // compilation.hooks.log.tap(pluginName, (name: any, entry: any) => {
+      //   if (entry.type === 'time') {
+      //     const compilerSpan = spans.get(compiler)!
+
+      //     // Ensure endTime is BigInt in microseconds
+      //     const endTime = process.hrtime.bigint()
+      //     const [label, hrtimeHigh, hrtimeLow] = entry.args
+      //     // console.log({ label, hrtimeHigh, hrtimeLow })
+      //     const duration = BigInt(String(hrtimeHigh) + String(hrtimeLow))
```
Comment (sokra) [inline packages/next/build/webpack/plugins/profiling-plugin.ts:121]: This makes no sense...

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 5bc3b2e4522171e2227f6bd60d42563ee5c5124e

**test/integration/config/test/index.test.js** 
- Configuration > should log webpack version correctly
- Configuration > should disable X-Powered-By header support
- Configuration > renders css imports
- Configuration > renders non-js imports from node_modules
- Configuration > renders server config on the server only
- Configuration > renders public config on the server only
- Configuration > renders the build id in development mode
- Configuration > correctly imports a package that defines `module` but no `main` in package.json
- Configuration > should have config available on the client
- Configuration > should update css styles using hmr
- Configuration > should update sass styles using hmr

<details>
<summary>Expand output</summary>

  ● Configuration › should log webpack version correctly

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › should disable X-Powered-By header support

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › renders css imports

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › renders non-js imports from node_modules

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › renders server config on the server only

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › renders public config on the server only

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › renders the build id in development mode

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › correctly imports a package that defines `module` but no `main` in package.json

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › should have config available on the client

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › should update css styles using hmr

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › should update sass styles using hmr

    FetchError: request to http://localhost:44387/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44387

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

</details>

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 7be44c5286694e895b3cce40ef54f5178d71b5c3

**test/integration/config/test/index.test.js** 
- Configuration > should log webpack version correctly
- Configuration > should disable X-Powered-By header support
- Configuration > renders css imports
- Configuration > renders non-js imports from node_modules
- Configuration > renders server config on the server only
- Configuration > renders public config on the server only
- Configuration > renders the build id in development mode
- Configuration > correctly imports a package that defines `module` but no `main` in package.json
- Configuration > should have config available on the client
- Configuration > should update css styles using hmr
- Configuration > should update sass styles using hmr

<details>
<summary>Expand output</summary>

  ● Configuration › should log webpack version correctly

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › should disable X-Powered-By header support

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › renders css imports

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › renders non-js imports from node_modules

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › renders server config on the server only

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › renders public config on the server only

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › renders the build id in development mode

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › correctly imports a package that defines `module` but no `main` in package.json

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › should have config available on the client

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › should update css styles using hmr

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

  ● Configuration › should update sass styles using hmr

    FetchError: request to http://localhost:44957/next-config failed, reason: connect ECONNREFUSED 127.0.0.1:44957

      at ClientRequest.<anonymous> (../node_modules/node-fetch/lib/index.js:1461:11)

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| buildDuration | 15.3s | 15.8s | ⚠️ +559ms |
| buildDurationCached | 3.9s | 3.7s | -151ms |
| nodeModulesSize | 178 MB | 178 MB | ⚠️ +2.47 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 3.157 | 3.164 | ⚠️ +0.01 |
| / avg req/sec | 791.95 | 790.12 | ⚠️ -1.83 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.632 | 1.744 | ⚠️ +0.11 |
| /error-in-render avg req/sec | 1531.66 | 1433.67 | ⚠️ -97.99 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 26.6 kB | 26.6 kB | ✓ |
| webpack-HASH.js gzip | 1.45 kB | 1.45 kB | ✓ |
| Overall change | 70.4 kB | 70.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| polyfills-a4..dd70.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| _app-HASH.js gzip | 979 B | 979 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.67 kB | 2.67 kB | ✓ |
| head-HASH.js gzip | 351 B | 351 B | ✓ |
| hooks-HASH.js gzip | 918 B | 918 B | ✓ |
| image-HASH.js gzip | 4.14 kB | 4.14 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 318 B | 318 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 13 kB | 13 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| _buildManifest.js gzip | 492 B | 492 B | ✓ |
| Overall change | 492 B | 492 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| index.html gzip | 539 B | 539 B | ✓ |
| link.html gzip | 552 B | 552 B | ✓ |
| withRouter.html gzip | 533 B | 533 B | ✓ |
| Overall change | 1.62 kB | 1.62 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Default Build with SWC</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| buildDuration | 8s | 8.2s | ⚠️ +216ms |
| buildDurationCached | 3.9s | 3.9s | -21ms |
| nodeModulesSize | 178 MB | 178 MB | ⚠️ +2.47 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 3.182 | 3.259 | ⚠️ +0.08 |
| / avg req/sec | 785.67 | 767.12 | ⚠️ -18.55 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.584 | 1.742 | ⚠️ +0.16 |
| /error-in-render avg req/sec | 1577.96 | 1435.14 | ⚠️ -142.82 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| 264.HASH.js gzip | 179 B | 179 B | ✓ |
| 675-HASH.js gzip | 13.8 kB | 13.8 kB | ✓ |
| framework-HASH.js gzip | 50.7 kB | 50.7 kB | ✓ |
| main-HASH.js gzip | 34.4 kB | 34.4 kB | ✓ |
| webpack-HASH.js gzip | 1.65 kB | 1.65 kB | ✓ |
| Overall change | 101 kB | 101 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| polyfills-a4..dd70.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| _app-HASH.js gzip | 1.33 kB | 1.33 kB | ✓ |
| _error-HASH.js gzip | 179 B | 179 B | ✓ |
| amp-HASH.js gzip | 331 B | 331 B | ✓ |
| css-HASH.js gzip | 352 B | 352 B | ✓ |
| dynamic-HASH.js gzip | 2.8 kB | 2.8 kB | ✓ |
| head-HASH.js gzip | 356 B | 356 B | ✓ |
| hooks-HASH.js gzip | 652 B | 652 B | ✓ |
| image-HASH.js gzip | 575 B | 575 B | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 2.2 kB | 2.2 kB | ✓ |
| routerDirect..HASH.js gzip | 327 B | 327 B | ✓ |
| script-HASH.js gzip | 392 B | 392 B | ✓ |
| withRouter-HASH.js gzip | 323 B | 323 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 10.2 kB | 10.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| _buildManifest.js gzip | 515 B | 515 B | ✓ |
| Overall change | 515 B | 515 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| index.html gzip | 539 B | 539 B | ✓ |
| link.html gzip | 551 B | 551 B | ✓ |
| withRouter.html gzip | 530 B | 530 B | ✓ |
| Overall change | 1.62 kB | 1.62 kB | ✓ |

</details>

</details>
Commit: 7be44c5286694e895b3cce40ef54f5178d71b5c3

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Build</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| buildDuration | 12.2s | 12.4s | ⚠️ +244ms |
| buildDurationCached | 3.1s | 3.1s | -15ms |
| nodeModulesSize | 182 MB | 182 MB | ⚠️ +2.58 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.612 | 2.624 | ⚠️ +0.01 |
| / avg req/sec | 957.18 | 952.57 | ⚠️ -4.61 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.289 | 1.306 | ⚠️ +0.02 |
| /error-in-render avg req/sec | 1939.59 | 1914.47 | ⚠️ -25.12 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| 745.HASH.js gzip | 179 B | 179 B | ✓ |
| framework-HASH.js gzip | 42.2 kB | 42.2 kB | ✓ |
| main-HASH.js gzip | 26.6 kB | 26.6 kB | ✓ |
| webpack-HASH.js gzip | 1.45 kB | 1.45 kB | ✓ |
| Overall change | 70.4 kB | 70.4 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| polyfills-a4..dd70.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| _app-HASH.js gzip | 979 B | 979 B | ✓ |
| _error-HASH.js gzip | 194 B | 194 B | ✓ |
| amp-HASH.js gzip | 312 B | 312 B | ✓ |
| css-HASH.js gzip | 329 B | 329 B | ✓ |
| dynamic-HASH.js gzip | 2.67 kB | 2.67 kB | ✓ |
| head-HASH.js gzip | 351 B | 351 B | ✓ |
| hooks-HASH.js gzip | 918 B | 918 B | ✓ |
| image-HASH.js gzip | 4.14 kB | 4.14 kB | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 1.66 kB | 1.66 kB | ✓ |
| routerDirect..HASH.js gzip | 318 B | 318 B | ✓ |
| script-HASH.js gzip | 387 B | 387 B | ✓ |
| withRouter-HASH.js gzip | 320 B | 320 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 13 kB | 13 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| _buildManifest.js gzip | 492 B | 492 B | ✓ |
| Overall change | 492 B | 492 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| index.html gzip | 540 B | 540 B | ✓ |
| link.html gzip | 552 B | 552 B | ✓ |
| withRouter.html gzip | 534 B | 534 B | ✓ |
| Overall change | 1.63 kB | 1.63 kB | ✓ |

</details>

</details>
<hr/>
<details>
<summary><strong>Default Build with SWC</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| buildDuration | 6.4s | 6.4s | ⚠️ +28ms |
| buildDurationCached | 3.2s | 3.1s | -104ms |
| nodeModulesSize | 182 MB | 182 MB | ⚠️ +2.58 kB |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall decrease ⚠️</summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.655 | 2.712 | ⚠️ +0.06 |
| / avg req/sec | 941.72 | 921.98 | ⚠️ -19.74 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.335 | 1.337 | 0 |
| /error-in-render avg req/sec | 1872.79 | 1869.39 | ⚠️ -3.4 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| 264.HASH.js gzip | 179 B | 179 B | ✓ |
| 675-HASH.js gzip | 13.8 kB | 13.8 kB | ✓ |
| framework-HASH.js gzip | 50.7 kB | 50.7 kB | ✓ |
| main-HASH.js gzip | 34.4 kB | 34.4 kB | ✓ |
| webpack-HASH.js gzip | 1.65 kB | 1.65 kB | ✓ |
| Overall change | 101 kB | 101 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| polyfills-a4..dd70.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| _app-HASH.js gzip | 1.33 kB | 1.33 kB | ✓ |
| _error-HASH.js gzip | 179 B | 179 B | ✓ |
| amp-HASH.js gzip | 331 B | 331 B | ✓ |
| css-HASH.js gzip | 352 B | 352 B | ✓ |
| dynamic-HASH.js gzip | 2.8 kB | 2.8 kB | ✓ |
| head-HASH.js gzip | 356 B | 356 B | ✓ |
| hooks-HASH.js gzip | 652 B | 652 B | ✓ |
| image-HASH.js gzip | 575 B | 575 B | ✓ |
| index-HASH.js gzip | 261 B | 261 B | ✓ |
| link-HASH.js gzip | 2.2 kB | 2.2 kB | ✓ |
| routerDirect..HASH.js gzip | 327 B | 327 B | ✓ |
| script-HASH.js gzip | 392 B | 392 B | ✓ |
| withRouter-HASH.js gzip | 323 B | 323 B | ✓ |
| bb14e60e810b..30f.css gzip | 125 B | 125 B | ✓ |
| Overall change | 10.2 kB | 10.2 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| _buildManifest.js gzip | 515 B | 515 B | ✓ |
| Overall change | 515 B | 515 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | timneutkens/next.js add/development-tracing-improvements | Change |
| - | - | - | - |
| index.html gzip | 538 B | 538 B | ✓ |
| link.html gzip | 551 B | 551 B | ✓ |
| withRouter.html gzip | 530 B | 530 B | ✓ |
| Overall change | 1.62 kB | 1.62 kB | ✓ |

</details>

</details>
Commit: ca88bd17e27d64298a5dbc923caec3f96284443f

**Review by sokra — APPROVED** [review]

**Pull request merged**